pyproject.toml

```diff
@@ -24,7 +24,7 @@ disallow_untyped_decorators = false
 disallow_untyped_defs = false
 ignore_missing_imports = true
 strict_equality = true
-strict_optional = true
+strict_optional = false
```

Comment:
We should create an issue to handle this before merging this PR so we don't lose track of it.

Reply:
Made a personal note, issue now added.